src/components/radios/_macro.njk

```diff
@@ -64,6 +65,15 @@
                 {% endif %}
             {% endfor %}
         </div>
+         {% if params.clear %}
+            {{ onsButton({
+                "text": params.clear.text,
+                "url": params.clear.url,
+                "type": "button",
+                "classes": "btn--secondary btn--small js-clear-btn u-mt-s u-db-no-js_enabled"
+            }) }}
+            <span class="js-clear-radio-alert u-vh" role="alert" aria-live="polite" data-clear="{{ params.clear.ariaClearText }}" data-cleared="{{ params.clear.ariaClearedText }}"></span>
+        {% endif %}
```

Comment:
I think we need to break the rules for this link/button and remove the CTA chevron because it's acting on a form element and not taking the user anywhere.

Reply:
ok